english/covid-19/WIP.md

```diff
@@ -2,3 +2,8 @@
 
 If you have something to add here then you can help us by [contributing to this dungeon](https://github.com/MakeContributions/markdown-dungeon/issues/112) 🙂
 
+
+As you are reading through the book, you hear a soft rumbling noice to your right. You are anxious as no one else is in the room. As you look slowly to your right, you find the Cloak of Invisibility out of the blue. Utterly confused, but you two choice now.
+
+1. wear the cloak and enter the dungeon
```

Comment:
Hi @parths239,
This is not the correct file to make this change, due to this for the WIP file